xmind/tests/test_workbook_saver.py

```diff
@@ -0,0 +1,135 @@
+import logging
+from . import base
+from unittest.mock import patch, MagicMock, mock_open
+from xmind.core.saver import WorkbookSaver
+from xmind.core.const import CONTENT_XML, XMIND_EXT
+
+
+class WorkbookSaverTest(base.Base):
+
+    def setUp(self):
+        super(WorkbookSaverTest, self).setUp()
+        self._workbook = MagicMock()
+        self._test_el = WorkbookSaver(self._workbook)
+
+    def getLogger(self):
+        if not getattr(self, '_logger', None):
+            self._logger = logging.getLogger('WoorkbookSaverTest')
+        return self._logger
+
+    def test_excessive_parameters(self):
+        # _workbook = MagicMock()
+        # _element = WorkbookSaver(_workbook)
+        _parameters = [
+            ('_get_content', 0),
+          #  ('save', 1)
+        ]
+
+        with self.assertRaises(TypeError) as _ex:
+            self._test_el.save(12,13)
+
+        self.getLogger().debug("Got an exception: '%s'", _ex.exception.args[0])
+        self.assertTrue(_ex.exception.args[0].find(
+            "%s() takes" % 'save') != -1)
+
+        for pair in _parameters:
+            with self.subTest(pair=pair):
+                self._test_method_by_excessive_parameters(pair, self._test_el)
+
+    def test_WoorkbookSaver_inint(self):
```

Comment:
mistypo ```_inint``` -> ```_init```

Reply:
btw, I think you can use approach which @vlad-shulika proposed to put 
```
_parameters = [
            ('_get_content', 0),
            ('save', (1, False)),
            ('__init__', 1)
        ]
``` 
into ```test_excessive_parameters``` and it will cover ```__init__``` as well

Reply:
Done